src/views/Profile.js

```diff
@@ -470,32 +381,12 @@ const ProfileTrustButton = ({ address, trustStatus }) => {
         onSuccess={handleRevokeTrustSuccess}
       />
       {!isOrganization && !isOwnAccount && (
-        <IconButton
-          classes={{
-            root: clsx(classes.trustButton, {
-              [classes.trustButtonIsMeTrusting]: trustStatus.isMeTrusting,
-              [classes.trustButtonIsTrustingMe]: trustStatus.isTrustingMe,
-              [classes.trustButtonMutuallyTrusted]:
-                trustStatus.isMeTrusting && trustStatus.isTrustingMe,
-            }),
-            disabled: classes.trustButtonDisabled,
-          }}
-          disabled={trustStatus.isPending || !isReady}
-          onClick={
-            trustStatus.isMeTrusting ? handleRevokeTrustOpen : handleTrustOpen
-          }
-        >
-          {trustStatus.isPending ? (
-            <CircularProgress size={24} />
-          ) : (
-            <Box className={classes.trustButtonContainer}>
-              <IconTrustCustomShape
-                className={classes.trustButtonIconBackground}
-              />
-              <TrustIcon className={classes.trustButtonIcon} />
-            </Box>
-          )}
-        </IconButton>
+        <UnevenRoundButton
```

Comment:
Now this UnevenRoundbutton is only for trust.

What I meant was to make a wrapper for IconButton which aways has the uneven round shape inseadof round button. If you look in the designs, I think you will see that this shape is also to be used for send button for example

Reply:
It is probably a good idea to update send icon button in this PR as well